src/components/editor/use-editor-setup.js

```diff
@@ -24,20 +21,11 @@ export function useEditorSetup( post ) {
 		receiveEntityRecords( 'postType', post.type, post );
 
 		setupEditor( post, {} );
-		registerCoreBlocks();
```

Comment:
This was relocated to the entry file as the blocks must be registered before we parse the raw post content provided by the host app. 

https://github.com/wordpress-mobile/GutenbergKit/blob/8ccd1fb448d6691fa1f1ba940657dd241c04790c/src/components/editor/use-host-bridge.js#L28